contracts/BadgerYieldSource.sol

```diff
@@ -11,14 +13,15 @@ import "hardhat/console.sol";
 /// @title A pooltogether yield source for badger sett
 /// @author Steffel Fenix, 0xkarl
 contract BadgerYieldSource is IYieldSource {
+    using SafeERC20 for IERC20;
     using SafeMath for uint256;
     IBadgerSett private immutable badgerSett;
-    IBadger private immutable badger;
+    IERC20 private immutable badger;
     mapping(address => uint256) private balances;
 
-    constructor(address badgerSettAddr, address badgerAddr) public {
-        badgerSett = IBadgerSett(badgerSettAddr);
-        badger = IBadger(badgerAddr);
+    constructor(IBadgerSett badgerSettAddr, IERC20 badgerAddr) public {
+        badgerSett = badgerSettAddr;
+        badger = badgerAddr;
```

Comment:
Would be more readable if it was written this way:
```suggestion
    constructor(IBadgerSett _badgerSettAddr, IERC20 _badgerAddr) public {
        badgerSett = _badgerSettAddr;
        badger = _badgerAddr;
```

Reply:
I like that.  It's also consistent with our convention of using _ for function args